docs/tutorial/part03.rst

```diff
@@ -2,5 +2,245 @@
+.. image:: images/relfrom.png
+   :width: 100%
+   :alt: alternate text
+
+
+Repeat these steps for each node class you wish to create. On this program those would be: ``address.py`` , ``intermediary.py`` , ``officer.py``, and ``other.py``. You must add the following code to each of the files:
+
+address.py ::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipFrom
+    )
+
+    class Address(StructuredNode):
+        sourceID       = StringProperty()
+        country_codes  = StringProperty()
+        valid_until    = StringProperty()
+        address        = StringProperty()
+        countries      = StringProperty()
+        node_id        = StringProperty()
+        officers       = RelationshipFrom('.officer.Officer', 'REGISTERED_ADDRESS')
+        intermediaries = RelationshipFrom('.intermediary.Intermediary', 'REGISTERED_ADDRESS')   
+
+
+intermediary.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo
+    )
+
+    class Intermediary(StructuredNode, NodeUtils):
+        sourceID      = StringProperty()
+        valid_until   = StringProperty()
+        name          = StringProperty()
+        country_codes = StringProperty()
+        countries     = StringProperty()
+        node_id       = StringProperty()
+        status        = StringProperty()
+        entities      = RelationshipTo('.entity.Entity', 'INTERMEDIARY_OF')
+        addresses     = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+
+
+
+officer.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo,
+    )
+
+    class Officer(StructuredNode):
+        sourceID      = StringProperty()
+        name          = StringProperty()
+        country_codes = StringProperty()
+        valid_until   = StringProperty()
+        countries     = StringProperty()
+        node_id       = StringProperty()
+        addresses     = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+        entities      = RelationshipTo('.entity.Entity', 'OFFICER_OF')
+
+other.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo,
+    )
+
+    class Other(StructuredNode):
+        sourceID    = StringProperty()
+        name        = StringProperty()
+        valid_until = StringProperty()
+        node_id     = StringProperty()
+        countries   = StringProperty()
+        addresses   = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+
+Modify the __init__.py file:
+==================================
+Now we need to add a little bit of code to the __init__.py. If you noticed in the previous code, we refer to other classes but we did no import them. e.g : ``.officer.Officer``. This means that it should not work because the program doesn't know what that is; Therefore, we are going to add all the imports to the init file so that each model have all the imports ::

```

Comment:
Just to know. Is `that each model have` or `that each model has`?
Sorry just to know 😅